hail/src/main/scala/is/hail/expr/ir/Compile.scala

```diff
@@ -230,19 +230,14 @@ object CompileWithAggregators2 {
     val fb = new EmitFunctionBuilder[F](argTypeInfo, GenericTypeInfo[R]())
 
     var ir = body
-    if (optimize)
```

Comment:
unnecessary I believe, optimize happens in lowering pipeline

Reply:
yep